===================================================================
RCS file: /usr/OpenBSD/cvs/src/usr.bin/ssh/channels.c,v
retrieving revision 1.170
retrieving revision 1.171
diff -u -r1.170 -r1.171
--- src/usr.bin/ssh/channels.c	2002/02/27 21:23:13	1.170
+++ src/usr.bin/ssh/channels.c	2002/03/04 19:37:58	1.171
@@ -146,7 +146,7 @@
 {
 	Channel *c;
 
-	if (id < 0 || id > channels_alloc) {
+	if (id < 0 || id >= channels_alloc) {
 		log("channel_lookup: %d: bad id", id);
 		return NULL;
 	}
